apps/docs/src/basics/colors.md

```diff
@@ -0,0 +1,103 @@
+
+### Global color scheme
+
+In order to establish a versatile foundation that offers to both light and dark modes, onyx colors are based on a comprehensive global color palette designed to function as a universally adaptable scheme. This palette spans from the lightest luminosity to the deepest darkness, covering a broad spectrum. However, it's important to note that the extremes at both ends of the palette are not suitable for all modes. Specifically, the delineation is set by values of 100 and 1200, demarcating the boundaries for usability in different modes. It's important to know that this nuanced palette of global shades is intentionally excluded from the design and development process of creating an onyx user interface.
+
+### Semantic color scheme
+
+The color values within the global color scheme have been precisely translated into the semantic context of light and dark modes. This refinement involves condensing the scale to a series of 9 values in increments of 100, with the pivotal point of 500 serving as the default reference for both light and dark modes. Notably, the semantic color schemes for the light mode now referres to a distinct spectrum when compared to their dark mode counterparts.
+
+Semantic colors are defined in relation to the background, dynamically adjusting luminance to harmonize with the unique characteristics of each mode. This intentional differentiation ensures that the visual aesthetics of the interface remain optimal in both light and dark settings.
+
+### Example
+
+![Image](/assets/color_systematic.png)
+
+## Themes
+
+Beyond the different shades, the hues in a color scheme also significantly impact the overall appearance. onyx offers two main types of color palettes: "themed color palettes" and "universal color palettes." Themed colors set the brand's tone in the UI, defining its unique look. Universal colors, on the other hand, support system information and are used across all themes to complete the brand's color range. This thoughtful approach ensures a good balance between expressing the brand's style and maintaining clarity and functionality for users.
+
+_color palette component by dev_
+
+To have access to the color token set for your implementation, please visit the [token documentation](/tokens/colors).
+
+## Roles of themed colors
+
+Themed colors reflect the brand's identity, adapting their main colors. While primary and secondary colors vary between the themes, the usage of them is constant.
+
+Primary colors highlight call-to-action elements and direct the focus of the user to the most important interactions. Brand-neutrals are used in the background and secondary actions, providing a subtle backdrop that allows primary colors to shine.
```

Comment:
That's the point with the naming to action and brand instead of primary and secondary. I cannot give a precise description here, because we are still pending. We have to maintain the docu afterwards.